packages/server/lib/utils/asyncWrapper.ts

```diff
@@ -10,7 +10,7 @@ export function asyncWrapper<TEndpoint extends Endpoint<any>, Locals extends Rec
         res: Response<TEndpoint['Reply'], Locals>,
         next: NextFunction
     ) => Promise<void> | void
-): RequestHandler<any, TEndpoint['Reply'], TEndpoint['Body'], TEndpoint['Querystring'], Locals> {
+): RequestHandler<any, TEndpoint['Reply'], any, any, any> {
```

Comment:
can you explain why this change is necessary? thank you

Reply:
Yes sorry, e.g: I was setting `never` for Body but it becomes unacceptable when a function expects `any`, since we don't really use chaining, the correctness of the return value is not relevant (but it's annoying :/)